src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiHplManifestSpec.groovy

```diff
@@ -15,6 +15,7 @@ class JpiHplManifestSpec extends Specification {
             apply plugin: 'jpi'
         }
         def libraries = [
+                new File(project.rootDir, 'src/main/resources'),
                 new File(project.buildDir, 'classes/java/main'),
                 new File(project.buildDir, 'resources/main'),
```

Comment:
Is there a test which shows that the libraries are added as well?

Reply:
Good point. I added it to the test and also mad sure that `afterEvaluate {}`s are triggered. Which was the problem why the test did not catch the regression before.